### PR TITLE
chore(doc): Update outdated diagram image

### PR DESCRIPTION
**Pull request by ariefrahmansyah**

### CODE REVIEW

**Comment by deadlycoconuts** [comment]
What happened to the merlin ui box in `docs/diagrams/architecture.drawio.svg` 🥲

**Comment by ariefrahmansyah** [comment]
> What happened to the merlin ui box in docs/diagrams/architecture.drawio.svg 🥲

The React UI application is served by Golang application, using this Dockerfile: https://github.com/caraml-dev/merlin/blob/main/Dockerfile

**Comment by krithika369** [comment]
> What happened to the merlin ui box in `docs/diagrams/architecture.drawio.svg` 🥲 Screenshot 2023-04-13 at 2 54 40 PM

@deadlycoconuts I guess the API and UI are merged into one box since the UI is served by the API?

@ariefrahmansyah, on this question - should we represent the react app in the Merlin box somehow to make it clear?

**Comment by deadlycoconuts** [comment]
Oops my bad, I was reading the architecture diagramme in a less technical way, and more of a diagramme showing different ways the components interact so the missing UI box looked scary.

**Comment by ariefrahmansyah** [comment]
Thanks @krithika369 for the review. I've updated it with your suggestion:

1. Add MLP box
2. Add react icon to Merlin box

File: docs/diagrams/architecture.drawio.svg
Comment (ariefrahmansyah) [inline docs/diagrams/architecture.drawio.svg:1]: Seems like there's something wrong with my drawio's vscode extension. In my vscode, I can see react icon, but not in GitHub :/

File: docs/reference/e2e-test.md [hunk 1]
```diff
@@ -4,36 +4,33 @@ Run E2E test as part of github actions workflow. This E2E test will be triggered
...
-* Setup python 3.7
-* Setup cluster
-* Setup mlp namespace
-* Deploy mlp
-* Deploy merlin
-* Run E2E test
-
+- Pull merlin repository
+- Pull mlp repository
+- Setup go 1.17
```
Comment (krithika369) [inline docs/reference/e2e-test.md:13]: Oops missed this earlier, we have Go 1.20 now, right? :) https://github.com/caraml-dev/merlin/blob/main/api/go.mod#L3

**Review by krithika369 — APPROVED** [review]

**Pull request merged**